src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -67,7 +67,7 @@ public ReadsSparkSource(JavaSparkContext ctx) {
             SAMRecord sam = v1._2().get();
             if (samRecordOverlaps(sam, intervals)) {
```

Comment:
out of curiosity, is `intervals` ever large?  Would this benefit from being broadcast?